modules/tinymce/src/core/test/ts/browser/dom/ElementTypeTest.ts

```diff
@@ -1,33 +1,36 @@
 import { describe, it } from '@ephox/bedrock-client';
-import { SugarElement } from '@ephox/sugar';
+import { SugarElement, SugarNode } from '@ephox/sugar';
 import { assert } from 'chai';
 
+import Schema from 'tinymce/core/api/html/Schema';
 import * as ElementType from 'tinymce/core/dom/ElementType';
 
 describe('browser.tinymce.core.dom.ElementTypeTest', () => {
   const checkElement = (name: string, predicate: (elm: SugarElement<Node>) => boolean, expectedValue: boolean) => {
     assert.equal(predicate(SugarElement.fromTag(name)), expectedValue, 'Should be the expected value for specified element');
   };
 
+  const baseSchema = Schema();
+
   const checkText = (predicate: (elm: SugarElement<Node>) => boolean) => {
     assert.isFalse(predicate(SugarElement.fromText('text')), 'Should be false for non element');
   };
 
   it('Check block elements', () => {
-    checkElement('p', ElementType.isBlock, true);
-    checkElement('h1', ElementType.isBlock, true);
-    checkElement('table', ElementType.isBlock, true);
-    checkElement('span', ElementType.isBlock, false);
-    checkElement('b', ElementType.isBlock, false);
-    checkText(ElementType.isBlock);
+    checkElement('p', (el) => baseSchema.isBlock(SugarNode.name(el)), true);
```

Comment:
These aren't really element type tests anymore, are they 😛

Reply:
you are right, I moved them to `SchemaTest`